src/ext/ddtrace.c

```diff
@@ -52,6 +52,8 @@ STD_PHP_INI_ENTRY("ddtrace.request_init_hook", "", PHP_INI_SYSTEM, OnUpdateStrin
                   zend_ddtrace_globals, ddtrace_globals)
 STD_PHP_INI_BOOLEAN("ddtrace.strict_mode", "0", PHP_INI_SYSTEM, OnUpdateBool, strict_mode, zend_ddtrace_globals,
                     ddtrace_globals)
+STD_PHP_INI_ENTRY("ddtrace.traced_internal_functions", "", PHP_INI_SYSTEM, OnUpdateString, traced_internal_functions,
```

Comment:
Any chance we can move this to an env var for now until we're able to get everything migrated over to INI settings at once? :) For the record, INI is totally where this should go, but until we get the post-install hook refactored, we're pretty much confined to env vars at the moment.

Reply:
Done.